meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-header.cmd.in

```diff
@@ -86,6 +86,19 @@ else
 	setenv fiovb.debug "${debug}"
 fi
 
+# Add a line that force enables rollback protection if
+# LMP_ROLLBACK_PROTECTION_ENABLE flag was set.
+
+if test "@@LMP_ROLLBACK_PROTECTION_ENABLE@@" = "1" && test "${fiovb.rollback_protection}" = "0"; then
```

Comment:
At runtime this evaluate to:
```
if test "0" = "1" && test "${fiovb.rollback_protection}" = "0"; then
```
imo having the variable are more readable, something like `fiovb.rollback_protection_enabled` for example::
```
setenv lmp_rollback_protection_enabled = "@@LMP_ROLLBACK_PROTECTION_ENABLE@@"
if test "${lmp_rollback_protection_enabled}" = "1" && test "${fiovb.rollback_protection}" = "0"; then
```

Reply:
fixed, thanks for suggestion!